components/learn/LearnHeader.vue

```diff
@@ -1,31 +1,31 @@
 <template>
-  <header class="textbook-beta-header">
-    <div class="textbook-beta-header__container-wrapper">
-      <div class="textbook-beta-header__container">
-        <div class="textbook-beta-header__logo-container">
-          <DynamicBackgroundLogo class="textbook-beta-header__logo" />
+  <header class="learn-header">
+    <div class="learn-header__container-wrapper">
+      <div class="learn-header__container">
+        <div class="learn-header__logo-container">
+          <DynamicBackgroundLogo class="learn-header__logo" />
         </div>
         <div>
-          <h1 class="textbook-beta-header__headline">
+          <h1 class="learn-header__headline">
             Qiskit Textbook (beta)
```

Comment:
should we also remove the `(beta)`  text?

Reply:
Hey @vabarbosa - not yet... if I remember, @JRussellHuffman had mentioned we'll still be in Beta for now...

Reply:
good to know.